test:mimxrt1010_evk:  tests/kernel/sched/schedule_api - kernel_threads_sched_userspace cases meet out our space

**Describe the bug**
kernel_threads_sched_userspace cases meet out our space

**To Reproduce**
Steps to reproduce the behavior:
tests/kernel/sched/schedule_api
1. mkdir build; cd build
2. cmake -DBOARD=mimxrt1010_evk ..
3. make
4. See error

**Expected behavior**
in former build this cases are PASS,but now the code size is too large

**Impact**
unknown

**Logs and console output**
```
+ docker exec confident_sinoussi build_zephyr_elf.sh mimxrt1010_evk_kernel3_master tests/kernel/sched/schedule_api mimxrt1010_evk build_900f0a3 -DCONF_FILE=prj_dumb.conf -DCONFIG_TIMESLICING=n kernel.scheduler.dumb_no_timeslicing tests/kernel/sched/schedule_api -DCONF_FILE=prj_dumb.conf -DCONFIG_TIMESLICING=n

/build/src/workspace/mimxrt1010_evk_kernel3_master

Including boilerplate (Zephyr base): /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/cmake/app/boilerplate.cmake

-- Application: /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api

-- Zephyr version: 2.5.99 (/build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr)

-- Found Python3: /usr/bin/python3.8 (found suitable exact version "3.8.5") found components: Interpreter 

-- Found west (found suitable version "0.9.0", minimum required is "0.7.1")

-- Board: mimxrt1010_evk

-- Cache files will be written to: /root/.cache/zephyr

-- Using toolchain: zephyr 0.12.2 (/opt/zephyr-sdk)

-- Found dtc: /opt/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.5.0", minimum required is "1.4.6")

-- Found BOARD.dts: /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/boards/arm/mimxrt1010_evk/mimxrt1010_evk.dts

-- Generated zephyr.dts: /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api/build_900f0a3/zephyr/zephyr.dts

-- Generated devicetree_unfixed.h: /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api/build_900f0a3/zephyr/include/generated/devicetree_unfixed.h

-- Generated device_extern.h: /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api/build_900f0a3/zephyr/include/generated/device_extern.h

Parsing /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/Kconfig

Loaded configuration '/build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/boards/arm/mimxrt1010_evk/mimxrt1010_evk_defconfig'

Merged configuration 'prj_dumb.conf'

Merged configuration '/build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api/build_900f0a3/zephyr/misc/generated/extra_kconfig_options.conf'

Configuration saved to '/build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api/build_900f0a3/zephyr/.config'

Kconfig header saved to '/build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api/build_900f0a3/zephyr/include/generated/autoconf.h'

-- The C compiler identification is GNU 10.2.0

-- The CXX compiler identification is GNU 10.2.0

-- The ASM compiler identification is GNU

-- Found assembler: /opt/zephyr-sdk/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc

-- Configuring done

-- Generating done

-- Build files have been written to: /build/src/workspace/mimxrt1010_evk_kernel3_master/zephyr/tests/kernel/sched/schedule_api/build_900f0a3

Scanning dependencies of target parse_syscalls_target

[  0%] Generating misc/generated/syscalls.json, misc/generated/struct_tags.json

[  1%] Built target parse_syscalls_target

Scanning dependencies of target syscall_list_h_target

[  2%] Generating include/generated/syscall_dispatch.c, include/generated/syscall_list.h

[  2%] Built target syscall_list_h_target

Scanning dependencies of target driver_validation_h_target

[  3%] Generating include/generated/driver-validation.h

[  3%] Built target driver_validation_h_target

Scanning dependencies of target kobj_types_h_target

[  4%] Generating include/generated/kobj-types-enum.h, include/generated/otype-to-str.h, include/generated/otype-to-size.h

[  4%] Built target kobj_types_h_target

Scanning dependencies of target offsets

[  4%] Building C object zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj

[  4%] Built target offsets

Scanning dependencies of target offsets_h

[  5%] Generating include/generated/offsets.h

[  5%] Built target offsets_h

Scanning dependencies of target zephyr_generated_headers

[  5%] Built target zephyr_generated_headers

Scanning dependencies of target app

[  5%] Building C object CMakeFiles/app.dir/src/main.c.obj

[  6%] Building C object CMakeFiles/app.dir/src/test_priority_scheduling.c.obj

[  7%] Building C object CMakeFiles/app.dir/src/test_sched_is_preempt_thread.c.obj

[  7%] Building C object CMakeFiles/app.dir/src/test_sched_priority.c.obj

[  8%] Building C object CMakeFiles/app.dir/src/test_sched_timeslice_and_lock.c.obj

[  8%] Building C object CMakeFiles/app.dir/src/test_sched_timeslice_reset.c.obj

[  9%] Building C object CMakeFiles/app.dir/src/test_slice_scheduling.c.obj

[  9%] Building C object CMakeFiles/app.dir/src/user_api.c.obj

[ 10%] Linking C static library app/libapp.a

[ 10%] Built target app

Scanning dependencies of target kernel

[ 10%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/device.c.obj

[ 11%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/errno.c.obj

[ 11%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/fatal.c.obj

[ 12%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/idle.c.obj

[ 12%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/init.c.obj

[ 13%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/kheap.c.obj

[ 13%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mailbox.c.obj

[ 14%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mem_slab.c.obj

[ 15%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/msg_q.c.obj

[ 15%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mutex.c.obj

[ 16%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/pipes.c.obj

[ 16%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/queue.c.obj

[ 17%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sched.c.obj

[ 17%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sem.c.obj

[ 18%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/stack.c.obj

[ 19%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/system_work_q.c.obj

[ 19%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/thread.c.obj

[ 20%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/version.c.obj

[ 20%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/condvar.c.obj

[ 21%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/work.c.obj

[ 21%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/smp.c.obj

[ 22%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/banner.c.obj

[ 22%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/xip.c.obj

[ 23%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/timeout.c.obj

[ 25%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/timer.c.obj

[ 25%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mempool.c.obj

[ 26%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/futex.c.obj

[ 26%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mem_domain.c.obj

[ 27%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/cache_handlers.c.obj

[ 27%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/userspace_handler.c.obj

[ 28%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/userspace.c.obj

[ 28%] Linking C static library libkernel.a

[ 28%] Built target kernel

Scanning dependencies of target zephyr

[ 28%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf.c.obj

[ 29%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_packaged.c.obj

[ 29%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc32c_sw.c.obj

[ 30%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc32_sw.c.obj

[ 31%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc16_sw.c.obj

[ 31%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc8_sw.c.obj

[ 32%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc7_sw.c.obj

[ 32%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/dec.c.obj

[ 33%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/fdtable.c.obj

[ 33%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/hex.c.obj

[ 34%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/notify.c.obj

[ 34%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/printk.c.obj

[ 35%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/onoff.c.obj

[ 36%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/rb.c.obj

[ 36%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/sem.c.obj

[ 37%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/thread_entry.c.obj

[ 37%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/timeutil.c.obj

[ 38%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/heap.c.obj

[ 38%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/heap-validate.c.obj

[ 39%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_complete.c.obj

[ 39%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/assert.c.obj

[ 40%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/mutex.c.obj

[ 41%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/user_work.c.obj

[ 41%] Building C object zephyr/CMakeFiles/zephyr.dir/misc/generated/configs.c.obj

[ 42%] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/nxp_imx/rt/soc.c.obj

[ 42%] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/log_minimal.c.obj

[ 43%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/console/uart_console.c.obj

[ 43%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/clock_control/clock_control_mcux_ccm.c.obj

[ 44%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/timer/sys_clock_init.c.obj

[ 44%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/timer/cortex_m_systick.c.obj

[ 45%] Linking C static library libzephyr.a

[ 45%] Built target zephyr

Scanning dependencies of target isr_tables

[ 46%] Building C object zephyr/arch/common/CMakeFiles/isr_tables.dir/isr_tables.c.obj

[ 47%] Linking C static library libisr_tables.a

[ 47%] Built target isr_tables

Scanning dependencies of target arch__common

[ 48%] Building C object zephyr/arch/common/CMakeFiles/arch__common.dir/sw_isr_common.c.obj

[ 48%] Linking C static library libarch__common.a

[ 48%] Built target arch__common

Scanning dependencies of target arch__arm__core__aarch32

[ 50%] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/swap.c.obj

[ 50%] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/swap_helper.S.obj

[ 51%] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/irq_manage.c.obj

[ 51%] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/thread.c.obj

[ 52%] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/cpu_idle.S.obj

[ 52%] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/fatal.c.obj

[ 53%] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/nmi.c.obj

[ 54%] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/nmi_on_reset.S.obj

[ 54%] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/prep_c.c.obj

[ 55%] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/isr_wrapper.S.obj

[ 55%] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/irq_offload.c.obj

[ 56%] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/userspace.S.obj

[ 56%] Linking C static library libarch__arm__core__aarch32.a

[ 56%] Built target arch__arm__core__aarch32

Scanning dependencies of target arch__arm__core__aarch32__cortex_m

[ 57%] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/vector_table.S.obj

[ 57%] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/reset.S.obj

[ 58%] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fault_s.S.obj

[ 59%] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fault.c.obj

[ 59%] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/exc_exit.S.obj

[ 60%] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/scb.c.obj

[ 60%] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/irq_init.c.obj

[ 61%] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/thread_abort.c.obj

[ 61%] Linking C static library libarch__arm__core__aarch32__cortex_m.a

[ 61%] Built target arch__arm__core__aarch32__cortex_m

Scanning dependencies of target arch__arm__core__aarch32__cortex_m__mpu

[ 62%] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/mpu/CMakeFiles/arch__arm__core__aarch32__cortex_m__mpu.dir/arm_core_mpu.c.obj

[ 63%] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/mpu/CMakeFiles/arch__arm__core__aarch32__cortex_m__mpu.dir/arm_mpu.c.obj

[ 63%] Linking C static library libarch__arm__core__aarch32__cortex_m__mpu.a

[ 63%] Built target arch__arm__core__aarch32__cortex_m__mpu

Scanning dependencies of target lib__libc__minimal

[ 63%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/abort.c.obj

[ 64%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/atoi.c.obj

[ 64%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtol.c.obj

[ 65%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoul.c.obj

[ 66%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/malloc.c.obj

[ 66%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/bsearch.c.obj

[ 67%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/exit.c.obj

[ 67%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strncasecmp.c.obj

[ 68%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strstr.c.obj

[ 68%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/string.c.obj

[ 69%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strspn.c.obj

[ 69%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/stdout_console.c.obj

[ 70%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/sprintf.c.obj

[ 71%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/fprintf.c.obj

[ 71%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/time/gmtime.c.obj

[ 72%] Linking C static library liblib__libc__minimal.a

[ 72%] Built target lib__libc__minimal

Scanning dependencies of target lib__posix

[ 72%] Building C object zephyr/lib/posix/CMakeFiles/lib__posix.dir/pthread_common.c.obj

[ 73%] Building C object zephyr/lib/posix/CMakeFiles/lib__posix.dir/nanosleep.c.obj

[ 73%] Linking C static library liblib__posix.a

[ 73%] Built target lib__posix

Scanning dependencies of target soc__arm__common__cortex_m

[ 75%] Building C object zephyr/soc/arm/common/cortex_m/CMakeFiles/soc__arm__common__cortex_m.dir/arm_mpu_regions.c.obj

[ 75%] Linking C static library libsoc__arm__common__cortex_m.a

[ 75%] Built target soc__arm__common__cortex_m

Scanning dependencies of target boards__arm__mimxrt1010_evk

[ 75%] Building C object zephyr/boards/arm/mimxrt1010_evk/CMakeFiles/boards__arm__mimxrt1010_evk.dir/pinmux.c.obj

[ 76%] Linking C static library libboards__arm__mimxrt1010_evk.a

[ 76%] Built target boards__arm__mimxrt1010_evk

Scanning dependencies of target subsys__testsuite__ztest

[ 77%] Building C object zephyr/subsys/testsuite/ztest/CMakeFiles/subsys__testsuite__ztest.dir/src/ztest.c.obj

[ 78%] Building C object zephyr/subsys/testsuite/ztest/CMakeFiles/subsys__testsuite__ztest.dir/src/ztest_error_hook.c.obj

[ 78%] Linking C static library libsubsys__testsuite__ztest.a

[ 78%] Built target subsys__testsuite__ztest

Scanning dependencies of target drivers__gpio

[ 78%] Building C object zephyr/drivers/gpio/CMakeFiles/drivers__gpio.dir/gpio_mcux_igpio.c.obj

[ 79%] Building C object zephyr/drivers/gpio/CMakeFiles/drivers__gpio.dir/gpio_handlers.c.obj

[ 79%] Linking C static library libdrivers__gpio.a

[ 79%] Built target drivers__gpio

Scanning dependencies of target drivers__serial

[ 80%] Building C object zephyr/drivers/serial/CMakeFiles/drivers__serial.dir/uart_mcux_lpuart.c.obj

[ 80%] Building C object zephyr/drivers/serial/CMakeFiles/drivers__serial.dir/uart_handlers.c.obj

[ 81%] Linking C static library libdrivers__serial.a

[ 81%] Built target drivers__serial

Scanning dependencies of target ..__modules__hal__nxp

[ 81%] Building C object modules/nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/devices/MIMXRT1011/fsl_clock.c.obj

[ 82%] Building C object modules/nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/boards/evkmimxrt1010/evkmimxrt1010_flexspi_nor_config.c.obj

[ 82%] Building C object modules/nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/drivers/imx/fsl_gpio.c.obj

[ 83%] Building C object modules/nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/drivers/imx/fsl_cache.c.obj

[ 83%] Building C object modules/nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/drivers/imx/fsl_lpuart.c.obj

[ 84%] Linking C static library lib..__modules__hal__nxp.a

[ 84%] Built target ..__modules__hal__nxp

Scanning dependencies of target app_smem_unaligned_linker

[ 85%] Generating app_smem_unaligned linker section

[ 85%] Built target app_smem_unaligned_linker

Scanning dependencies of target linker_app_smem_unaligned_script

[ 86%] Generating linker_app_smem_unaligned.cmd

[ 86%] Built target linker_app_smem_unaligned_script

Scanning dependencies of target app_smem_unaligned_prebuilt

[ 87%] Building C object zephyr/CMakeFiles/app_smem_unaligned_prebuilt.dir/misc/empty_file.c.obj

[ 87%] Linking C executable app_smem_unaligned_prebuilt.elf

Logical command for additional byproducts on target: app_smem_unaligned_prebuilt



[ 87%] Built target app_smem_unaligned_prebuilt

Scanning dependencies of target app_smem_aligned_linker

[ 87%] Generating app_smem_aligned linker section

[ 87%] Built target app_smem_aligned_linker

Scanning dependencies of target linker_zephyr_prebuilt_script_target

[ 88%] Generating linker_zephyr_prebuilt.cmd

[ 88%] Built target linker_zephyr_prebuilt_script_target

Scanning dependencies of target zephyr_prebuilt

[ 89%] Building C object zephyr/CMakeFiles/zephyr_prebuilt.dir/misc/empty_file.c.obj

[ 90%] Linking C executable zephyr_prebuilt.elf

Logical command for additional byproducts on target: zephyr_prebuilt



[ 90%] Built target zephyr_prebuilt

Scanning dependencies of target kobj_hash_list

[ 91%] Generating kobject_hash.gperf

[ 91%] Built target kobj_hash_list

Scanning dependencies of target kobj_hash_output_src_pre

[ 92%] Generating kobject_hash_preprocessed.c

[ 92%] Built target kobj_hash_output_src_pre

[ 93%] Generating kobject_hash.c

Scanning dependencies of target kobj_hash_output_lib

[ 93%] Building C object zephyr/CMakeFiles/kobj_hash_output_lib.dir/kobject_hash.c.obj

[ 94%] Linking C static library libkobj_hash_output_lib.a

[ 95%] Built target kobj_hash_output_lib

Scanning dependencies of target kobj_hash_output_obj_renamed

[ 95%] Generating kobject_hash_renamed.o

[ 95%] Built target kobj_hash_output_obj_renamed

Scanning dependencies of target linker_zephyr_final_script_target

[ 96%] Generating linker.cmd

[ 96%] Built target linker_zephyr_final_script_target

[ 96%] Generating dev_handles.c

[ 97%] Generating isr_tables.c, isrList.bin

Scanning dependencies of target zephyr_final

[ 98%] Building C object zephyr/CMakeFiles/zephyr_final.dir/misc/empty_file.c.obj

[ 98%] Building C object zephyr/CMakeFiles/zephyr_final.dir/isr_tables.c.obj

[100%] Building C object zephyr/CMakeFiles/zephyr_final.dir/dev_handles.c.obj

[100%] Linking C executable zephyr.elf

/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.2.0/../../../../arm-zephyr-eabi/bin/ld: zephyr.elf section `priv_stacks_noinit' will not fit in region `SRAM'

/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.2.0/../../../../arm-zephyr-eabi/bin/ld: region `SRAM' overflowed by 6880 bytes

Memory region         Used Size  Region Size  %age Used

           OCRAM:          0 GB        64 KB      0.00%

           FLASH:       94812 B        16 MB      0.57%

            ITCM:          0 GB        32 KB      0.00%

            SRAM:       39648 B        32 KB    121.00%

        IDT_LIST:          0 GB         2 KB      0.00%

collect2: error: ld returned 1 exit status

make[2]: *** [zephyr/CMakeFiles/zephyr_final.dir/build.make:146: zephyr/zephyr.elf] Error 1

make[1]: *** [CMakeFiles/Makefile2:2562: zephyr/CMakeFiles/zephyr_final.dir/all] Error 2

make: *** [Makefile:84: all] Error 2

script returned exit code 2
```

**Environment (please complete the following information):**
 - OS: (e.g. Linux, )
 - Toolchain (e.g Zephyr SDK, ...)
 - Commit SHA or Version used: zephyr-v2.5.0-1463-gc59cf6008b60

--- a/boards/arm/mimxrt1010_evk/mimxrt1010_evk.dts
+++ b/boards/arm/mimxrt1010_evk/mimxrt1010_evk.dts
@@ -18,7 +18,7 @@
 	};
 
 	chosen {
-		zephyr,sram = &dtcm;
+		zephyr,sram = &ocram;
 		zephyr,itcm = &itcm;
 		zephyr,console = &lpuart1;
 		zephyr,shell-uart = &lpuart1;

